settings.h

```diff
@@ -10,30 +10,27 @@ class Settings : public QObject
     Q_PROPERTY(QString installPath READ installPath WRITE setInstallPath NOTIFY installPathChanged)
     Q_PROPERTY(QString commandLine READ commandLine WRITE setCommandLine NOTIFY commandLineChanged)
     Q_PROPERTY(QString currentVersion READ currentVersion WRITE setCurrentVersion NOTIFY currentVersionChanged)
-    Q_PROPERTY(bool installFinished READ installFinished WRITE setInstallFinished NOTIFY installFinishedChanged)
 public:
     const QString INSTALL_PATH = "settings/installPath";
     const QString COMMAND_LINE = "settings/commandLineParameters";
     const QString CURRENT_VERSION = "settings/currentVersion";
-    const QString INSTALL_FINISHED = "settings/installFinished";
+    // const QString INSTALL_FINISHED = "settings/installFinished";  // No longer used
```

Comment:
delete then?

Reply:
I think it's good to keep track of what names were used in the past, so that one is not accidentally reused.